main.cpp

```diff
@@ -35,7 +35,7 @@ void printUsage() {
   printf("         inFile\n");
   printf(" -v: Print version\n");
   printf(" -gpu: Enable gpu calculation\n");
-  printf(" -gpu gpuId1,gpuId2,...: List of GPU(s) to use, default is 0\n");
+  printf(" -gpuId gpuId1,gpuId2,...: List of GPU(s) to use, default is 0\n");
   printf(" -g g1x,g1y,g2x,g2y,...: Specify GPU(s) kernel gridsize, default is 2*(MP),2*(Core/MP)\n");
   printf(" -d: Specify number of leading zeros for the DP method (default is auto)\n");
```

Comment:
I updated the typos.
I was unable to accept the pull request, too much mods in the code to try to merge with git.
Man thanks ;)

Reply:
Sorry, I tried to merge with your last changes and I broke my entire fork :'(
Thanks for your work ;)